### PR TITLE
Added unit tests

### PR DESCRIPTION
**Pull request by ValentinTyhonov**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -65,8 +70,33 @@
...
+			<artifactId>mockserver-netty</artifactId>
+			<version>5.3.0</version>
+			<scope>test</scope>
+		</dependency>
+		<!-- https://mvnrepository.com/artifact/org.mock-server/mockserver-client-java -->
+		<dependency>
+			<groupId>org.mock-server</groupId>
+			<artifactId>mockserver-client-java</artifactId>
+			<version>5.3.0</version>
+		</dependency>
```
Comment (tobiasake) [inline pom.xml]: These two new components to be FOSSed.

File: src/test/java/com/ericsson/ei/frontend/WebControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    }
+
+    @AfterClass
+    public static void afterClass() {
+        System.clearProperty("ei.frontendServiceHost");
+        System.clearProperty("ei.frontendServicePort");
+        System.clearProperty("ei.frontendContextPath");
+        System.clearProperty("ei.useSecureHttp");
+    }
+
```
Comment (tobiasake) [inline src/test/java/com/ericsson/ei/frontend/WebControllerTest.java]: You have missed to test getFrontendServiceUrl() function,, or has it been moved?

Comment (ValentinTyhonov) [reply src/test/java/com/ericsson/ei/frontend/WebControllerTest.java]: getFrontendServiceUrl() function is checked in every test here, when I check model().attribute("frontendServiceUrl", "http://localhost:9090/somePath")

Comment (tobiasake) [reply src/test/java/com/ericsson/ei/frontend/WebControllerTest.java]: ok

File: src/main/java/com/ericsson/ei/frontend/utils/BackEndInstancesUtils.java [hunk 3]
```diff
@@ -124,11 +124,7 @@ public boolean checkIfInstanceAlreadyExist(JsonObject instance) {
 
     public void writeIntoFile() {
         try {
```
Comment (tobiasake) [inline src/main/java/com/ericsson/ei/frontend/utils/BackEndInstancesUtils.java]: Ericssons copyright header seems to be missing in this file.

Comment (ValentinTyhonov) [reply src/main/java/com/ericsson/ei/frontend/utils/BackEndInstancesUtils.java]: No. Header was added by Andriy befor. So it exists here.

Comment (tobiasake) [reply src/main/java/com/ericsson/ei/frontend/utils/BackEndInstancesUtils.java]: Ok. But its no visible here. But if its there, then its ok.

**Pull request merged**